### ISSUE TITLE
Clothes Issue

### ISSUE THREAD

**Issue by 0x80249fff** [PainedPsyche/cui_character#74]
Yesterday was everything fine. Today seems to be not working anymore? Is something changed?

There is no clothes to select everything else is working tho.

![error1](https://user-images.githubusercontent.com/102218906/162590672-700b2e58-bddf-455d-ad44-5a1bff4cfb72.png)

After printing GetClothesData() function im getting emtpy table like this:

![error2](https://user-images.githubusercontent.com/102218906/162590736-84e8d175-d8be-4d51-92fd-da79606086b9.png)

**Comment by myStoryRP** [comment]
I also have this issue

**Comment by hugoshao** [comment]
I have the same problem, if know how to fix this, please let me know! thanks

**Comment by gitSNAKEgit** [comment]
Same issue here. Haven't touched this script in months and now clothing doesn't show up, either in character creation or clothing stores.

**Comment by Nywarda** [comment]
Hi, have you managed to fix it?

**Comment by gitSNAKEgit** [comment]
Not yet but, functions 'GetComponentDataFromHash' and 'GetPropDataFromHash' are both returning nil when they should be returning a table containing the clothing data, including the name of the item. Honestly I don't understand how these functions work.

**Comment by SeaLife** [comment]
Maybe something bricked by CFX? It worked on release-branch 3-4 days ago and it did not work on unstable..
Maybe they released this stuff now and bricked the release branch?

**Comment by gitSNAKEgit** [comment]
Yeah it looks like it. Which is a shame because this is an awesome script and I'd hate to have to use something else.

**Comment by LostJ3ster** [comment]
I made a pull request with a workaround for this. Instead of using the native, I'm accessing root-cause's repository with all the clothing information.
Everything else stays the same.
While my pull request is waiting for approval, feel free to use my fork.
